simpledotorg/simple-server
# Update ruby to 2.74

**Story card:** [ch3876](https://app.shortcut.com/simpledotorg/story/3876/upgrade-ruby-to-latest-2-7-release)

## Because

We want to upgrade our version of Ruby

## How to test

ensure you have ruby 2.7.4 installed, run `bundle` and `rails server`.  Verify the app works.
